src/Model/Profile.php

```diff
@@ -24,6 +24,8 @@
 use Friendica\Util\Temporal;
 use dba;
 
+use \InvalidArgumentException;
```

Comment:
Code Standards: Please remove leading `\` in `use` statements.

Reply:
Done.